geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStoreHelper.scala

```diff
@@ -0,0 +1,103 @@
+package org.locationtech.geomesa.kafka
+
+import java.util.UUID
+
+import org.locationtech.geomesa.utils.geotools.FeatureUtils
+import org.opengis.feature.simple.SimpleFeatureType
+import org.locationtech.geomesa.utils.geotools.Conversions.RichSimpleFeatureType
+
+object KafkaDataStoreHelper {
```

Comment:
The KafkaDataStoreHelper is responsible for managing the userdata in SFT required by KafkaDataStore. Its enough of its own thing that I'd like to keep it in its own file.  However, I'm not opposed to renaming it if you can suggest something better.